f/assignGear/f_assignGear_3IFB.sqf

```diff
@@ -166,6 +166,9 @@ _medkit = "Medikit";
 // Night Vision Goggles
 _nvg = "NVGoggles_OPFOR";
 
+// Binoculars or rangefinder
```

Comment:
Any reason why this comment should be different from the others?

Reply:
It's because I did that one first, then got distracted, then came back and did the rest. I'll change it.